bin/cml-pr.js

```diff
@@ -11,15 +11,17 @@ const CML = require('../src/cml');
 const run = async (opts) => {
   const globs = opts._.length ? opts._ : undefined;
   const cml = new CML(opts);
-  print(await cml.pr_create({ ...opts, globs }));
+  print((await cml.pr_create({ ...opts, globs })) || '');
 };
 
 const opts = decamelize(
   yargs
-    .strict()
```

Comment:
not strict?

Reply:
I don't fully remember why but it had to be with the multiple input... 🤔  Im putting it back and see...

Reply:
Seems to be fine